src/frontends/onnx/frontend/src/ops_bridge.hpp

```diff
@@ -77,6 +76,15 @@ class OperatorsBridge {
     void overwrite_operator(const std::string& name, const std::string& domain, Operator fn);
 
 private:
+    void register_operator_in_custom_domain(const std::string& name,
+                                            const VersionRange& range,
+                                            Operator fn,
+                                            const std::string& domain,
+                                            std::string warning_mes = "");
+    void register_operator(const std::string& name,
```

Comment:
Not sure but parameter name, range could be passed by copy as it looks like they are created from temporaries?

Reply:
done